src/controllers/active.directors.controller.ts

```diff
@@ -125,7 +124,7 @@ async function beginTerminationJourney(req: Request, res: Response, session: Ses
 /**
  * Post an officer filing and redirect to the first page in the CH01 journey.
 */
-async function beginUpdateJourney(req: Request, res: Response, session: Session, companyNumber: string, transactionId: string, appointmentId: any) {
+async function beginUpdateJourney(req: Request, res: Response, session: Session, companyNumber: string, transactionId: string, appointmentId: any, lang: string | undefined) {
```

Comment:
should we have an undefined type here? `en` is always the default case for selectLang().

Reply:
Removed altogether as not part of the ticket. However it should not be undefined as it will be required for the url query